release/serve_tests/workloads/serve_test_cluster_utils.py

```diff
@@ -63,14 +63,22 @@ def setup_anyscale_cluster(checkpoint_path: str = DEFAULT_CHECKPOINT_PATH):
 
 
 @ray.remote
-def warm_up_one_cluster(
-        num_warmup_iterations: int,
-        http_host: str,
-        http_port: str,
-        endpoint: str,
-) -> None:
+def warm_up_one_cluster(num_warmup_iterations: int,
+                        http_host: str,
+                        http_port: str,
+                        endpoint: str,
+                        nonblocking: bool = False) -> None:
+    timeout = 0.0001 if nonblocking else None
```

Comment:
Out of curiosity, what's the context for this number, is it just an arbitrary small number?

Reply:
Yep that's right. The `requests.get()` call is blocking by default– it waits for a response before it returns. With a `timeout`, it instead raises an exception after that time passes. Making the timeout small essentially turns the `requests.get()` call into a non-blocking operation (as long as you don't care about the return value).